lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -243,12 +243,32 @@ sub show {
     my ($self) = @_;
 
     return $self->reply->not_found if (!defined $self->param('testid'));
+    my $referer = $self->req->headers->header('Referer') // '';
 
     my $job = $self->app->schema->resultset("Jobs")->search(
         {
             id => $self->param('testid')
         },
         {prefetch => qw(jobs_assets)})->first;
+    $referer = Mojo::URL->new($referer)->host;
```

Comment:
Right. Forgot about that one. What about api route access? Originally I planned for that as well, but then why?